tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
Maybe I'm missing something, but instead of `invalid_names_list()` and single-use variable we can use `invalid_values_list()`, which includes all the invalid names provided by `invalid_names_list()`, + empty name, + space character, + tab character. And if some new invalid value is needed in future - then the helper can be updated, so we'll need to change 1 line instead of each test.